deploy-agent/README.md

```diff
@@ -1,7 +1,39 @@
 Deploy-agent is a python script runs on every host and execute deploy scripts.
 See https://github.com/pinterest/teletraan/wiki for more details.
 
-To run unit tests (debian comapatible):
+# Before you commit new code
+
+1. Install [pre-commit](https://pre-commit.com/#install)
+```bash
+pip install pre-commit
+pre-commit install
+```
```

Comment:
Since this applies to deploy-board and deploy-service too, how about putting it on the top-level README?

Reply:
Since documentation should be optimized for reads, I'm in favor of duplicating it in both places.